src/libraries/System.ComponentModel.Primitives/tests/TrimmingTests/VerifyCategoryNamesDontGetTrimmed.cs

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.ComponentModel;
+
+namespace Test
+{
+    /// <summary>
+    /// When UseSystemResourceStrings feature switch is on, we want to validate that getting the resource string of
+    /// a category attribute won't result in having "PropertyCategory" appended to the beginning of the resulting string.
+    /// This test ensures that both built-in categories as well as custom categories get the right Category when the
+    /// feature switch is on.
+    /// </summary>
+    public class Program
+    {
+        public static int Main()
+        {
+            if (GetEnumCategory(AnEnum.Action) == "Action" && GetEnumCategory(AnEnum.Something) == "Something")
```

Comment:
Can you add a test for one of the strings with a space in it? Like `WindowStyle`? That will ensure we aren't just returning the string back, but instead getting the real "resource" value.